auth/service_test.go

```diff
@@ -289,3 +293,690 @@ func TestIdentify(t *testing.T) {
 		assert.Equal(t, tc.idt, idt, fmt.Sprintf("%s expected %s got %s\n", tc.desc, tc.idt, idt))
 	}
 }
+
+func TestCreateGroup(t *testing.T) {
+	svc := newService()
+	_, secret, err := svc.Issue(context.Background(), "", auth.Key{Type: auth.UserKey, IssuedAt: time.Now(), IssuerID: id, Subject: email})
+	assert.Nil(t, err, fmt.Sprintf("Issuing login key expected to succeed: %s", err))
+
+	key := auth.Key{
+		ID:       "id",
```

Comment:
Maybe this should be a real ID?
Same for all other tests

Reply:
is it relevant, this is a test and not a part of this PR

Reply:
ok resolving this one if you want